.github/workflows/tests.yml

```diff
@@ -27,7 +27,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-20.04, ubuntu-22.04, windows-2019, macos-14]
+        os: [ubuntu-22.04, ubuntu-22.04, windows-2019, macos-14]
```

Comment:
```suggestion
        os: [ubuntu-22.04, windows-2019, macos-14]
```

Reply:
ah, good catch

Will manually fix, rather than take suggestion, or else you can't review

Reply:
or...

```suggestion
        os: [ubuntu-22.04, ubuntu-24.04, windows-2019, macos-14]
```